gsa/src/web/components/form/checkbox.js

```diff
@@ -49,19 +50,49 @@ const CheckboxComponent = ({
   toolTipTitle,
   unCheckedValue,
   ...other
-}) => (
-  <StyledElement>
-    <Divider title={toolTipTitle}>
-      <StyledInput {...other} disabled={disabled} type="checkbox" />
-      {isDefined(title) && (
-        <StyledTitle data-testid="checkbox-title" disabled={disabled}>
-          {title}
-        </StyledTitle>
-      )}
-      {children}
-    </Divider>
-  </StyledElement>
-);
+}) => {
+  const [value, setValue] = useState(undefined);
```

Comment:
Either we need to initialize value with the value from the prop or we drop the internal state completely. Dropping the internal state would make the component controlled only but that would be fine for me.

Reply:
> Dropping the internal state would make the component controlled only but that would be fine for me.

Sounds great to me!